vibe/src/main/java/com/vibeapp/vibe/controllers/AnnouncementController.java

```diff
@@ -33,6 +40,24 @@ public String getAllAnnouncements(Model model) {
         return "users/announcements";
     }
 
+    @Transactional
+    @PostMapping("/users/announcements")
+    public String getAllAnnouncements(@RequestParam Map<String, String> formData,Model model){
+        String username = formData.get("username");
+        Profile user = proRepo.findByName(username);
+        model.addAttribute("username", user);
+
+        System.out.println("Getting all announcements");
+        List<Announcement> announcements = announceRepo.findAllByOrderByAidAsc();

```

Comment:
```suggestion
        List<Announcement> announcements = announceRepo.findAllByOrderByAidDesc();

```
AidAsc() isnt an available method